docs/API_SPECS_SIMULATIONS.md

```diff
@@ -611,10 +611,45 @@ to ignore any change happened and overwrite the existing simulation.  Note
 that this is meant to be used only when data loss is acceptable, e.g.
 during development sessions and in test environments.
 
-## Deleting a simulation
+## Delete a simulation
 
 Simulations can be deleted using the DELETE method.
 
 ```
 DELETE /v1/simulations/1
-```
\ No newline at end of file
+```
+
+## Add device in a simulation
+
+Add a device to a simulatiom using AddDevice method.
+
+Request:
+```
+PUT /v1/simulations/1/AddDevice
+Content-Type: application/json; charset=utf-8
+```
+```json
+{
+  "Item": "truck-01.0",
```

Comment:
the payload should contain the device model at least.  The field "Item" (which would be "Id") is not required because the Device Id is in the URL